packages/babel-plugin-emotion/src/index.js

```diff
@@ -21,6 +21,20 @@ export function hashArray(arr) {
 }
 
 const staticStylis = new Stylis({ keyframe: false })
+/**
+ * Sets the prefix option for Stylis.
+ * @param {Object} opts - babel options
+ * @param {boolean|function} opts.prefix - should the rule be prefixed
+ */
+const setIsPrefixed = ({ prefix = true }) => {
+  if (typeof prefix === 'function') {
+    staticStylis.use({
```

Comment:
The [documentation](https://github.com/thysultan/stylis.js#vendor-prefixing) for this had a mistake, this should use `.set`